extensions/amp-story/amp-story.md

```diff
@@ -1386,6 +1386,177 @@ The `<amp-story-bookend>` must have a `src` attribute pointing to the JSON confi
+
+## Manipulating Navigation
+The features listed below allow you to manipulate navigation within a story. Note that when using branching features, navigation throughout the story from a 'branch-off point' will follow the path that a user takes. That is, if a user navigates from page-1 to page-5 and then page-6, navigating backwards will follow this exact path (page-6 to page-5 and finally page-1). The other pages will not be exposed to the user.
+
+### The `goToPage` Action
+A new `goToPage` action allows for 'jumping' from one page within a story to another. The action can be used with allowed elements in `amp-story` and their respective events. Clicking on an element with this action should bring the user to another page within the same story. Using this action **requires** that you give the `amp-story` element an id, a unique identifier, as the story itself is the target.  Read more about actions and events in AMP [here](https://amp.dev/documentation/guides-and-tutorials/learn/amp-actions-and-events.html?format=stories).
+
+
+<table>
+<thead>
+<tr>
+<th>Action</th>
+<th>Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>goToPage(id=STRING)</td>
+<td>Navigates to a target page. Id refers to the <a href="https://amp.dev/documentation/components/reference/amp-story.html?format=websites#id-[required]">required page id</a> </td>
+</tr>
+</tbody>
+</table>
+
+The following example shows a story with a quiz experience. The first page acts as a branch off point that allows the user to select the path that the story takes. <a name="quiz-example"></a>
+
+```html
+<amp-story id="story" standalone>
+    <amp-story-page id="cover">
+        <amp-story-grid-layer>
+        <p> What is your favourite food? </p>
+          <button on="tap:story.goToPage(id=bacon-answer)"> Bacon! </button>
+          <button on="tap:story.goToPage(id=boba-answer)"> Boba! </button>
+          <button on="tap:story.goToPage(id=pizza-answer)"> Pizza! </button>
+        </amp-story-grid-layer>
+    </amp-story-page>
+
+    <amp-story-page id="bacon-answer">
+        <amp-story-grid-layer>
+          <p> Bacon, of course! </p>
+        </amp-story-grid-layer>
+    </amp-story-page>
+
+    <amp-story-page id="boba-answer">
+        <amp-story-grid-layer>
+          <p> Boba, because it's a perfect drink! </p>
+        </amp-story-grid-layer>
+    </amp-story-page>
+
+    <amp-story-page id="pizza-answer">
+        <amp-story-grid-layer>
+        <p> Pizza, it's a true classic! </p>
+        </amp-story-grid-layer>
+    </amp-story-page>
+    ...
+</amp-story>
+```
+
+### The `advance-to` Attribute
+The `advance-to` attribute allows you to specify what page should follow a given `amp-story-page` when the user advances through the story. Recall that without branching the next page is by default the next `amp-story-page` in the DOM. This is still the default behavior should you choose not to use this attribute when creating non-linear stories.
```

Comment:
Update section to:
The `advance-to` attribute specifies which page should follow a given `amp-story-page` as the user advances through the story. Without branching, the following page defaults to the next `amp-story-page` in the DOM. This will still be the default behavior if you choose to not specify the `advance-to` attribute when creating non-linear stories.

Reply:
Done